.gitignore

```diff
@@ -84,3 +84,9 @@ myenv/
 node_modules/
 
 webpack-stats.json
+
+# Excel temp files
+~$*.xlsx
+
+# Auto-assembled media file
```

Comment:
this is a folder, but i think that's fine

Reply:
Um, right you are.  I might as well update the comment while I'm making other commits.